pkgs/epnix/support/twincat-ads/default.nix

```diff
@@ -23,6 +23,8 @@ mkEpicsPackage rec {
     sha256 = "sha256-f7hod1N1AzCh+W7nHl9VCA+nuwpJAboSh19Dq80n/2E=";
   };
 
+  patches = [./fix-missing-header.patch];
```

Comment:
I have writtent an issue to ESS for them to update the library, if you want to add a comment here mentionning it for further support:
https://gitlab.esss.lu.se/epics-modules/epics-twincat-ads/-/issues/2

Reply:
Thanks! Done.